<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.441" tests="2" failures="2">
  <testsuite name="Root Suite" timestamp="2025-01-15T08:37:03" tests="0" file="cypress\e2e\network\putRequest.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Requests: PUT" timestamp="2025-01-15T08:37:03" tests="2" time="0.437" failures="2">
    <testcase name="Requests: PUT should be able to update a players active-state correctly in the frontend and the backend. " time="0.000" classname="should be able to update a players active-state correctly in the frontend and the backend. ">
      <failure message="The following error originated from your application code, not from Cypress.

  &gt; Cannot read properties of null (reading &apos;isActive&apos;)

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application" type="TypeError"><![CDATA[TypeError: The following error originated from your application code, not from Cypress.

  > Cannot read properties of null (reading 'isActive')

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application
    at SelectedPlayer (http://localhost:4173/src/components/SelectedPlayer.jsx:22:51)
    at renderWithHooks (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:11548:26)
    at mountIndeterminateComponent (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:14926:21)
    at beginWork (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:15914:22)
    at HTMLUnknownElement.callCallback2 (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3674:22)
    at Object.invokeGuardedCallbackDev (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3699:24)
    at invokeGuardedCallback (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3733:39)
    at beginWork$1 (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19765:15)
    at performUnitOfWork (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19198:20)
    at workLoopSync (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19137:13)]]></failure>
    </testcase>
    <testcase name="Requests: PUT Should not update the player if update fails." time="0.000" classname="Should not update the player if update fails.">
      <failure message="The following error originated from your application code, not from Cypress.

  &gt; Cannot read properties of null (reading &apos;isActive&apos;)

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application" type="TypeError"><![CDATA[TypeError: The following error originated from your application code, not from Cypress.

  > Cannot read properties of null (reading 'isActive')

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application
    at SelectedPlayer (http://localhost:4173/src/components/SelectedPlayer.jsx:22:51)
    at renderWithHooks (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:11548:26)
    at mountIndeterminateComponent (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:14926:21)
    at beginWork (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:15914:22)
    at HTMLUnknownElement.callCallback2 (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3674:22)
    at Object.invokeGuardedCallbackDev (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3699:24)
    at invokeGuardedCallback (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:3733:39)
    at beginWork$1 (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19765:15)
    at performUnitOfWork (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19198:20)
    at workLoopSync (http://localhost:4173/node_modules/.vite/deps/chunk-NGODP64W.js?v=28d3b1ef:19137:13)]]></failure>
    </testcase>
  </testsuite>
</testsuites>